# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../dietlibc/pkg_patch/pkg_gpm.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- gpm-1.20.1/src/synaptics.c.vanilla	2005-02-07 11:41:12.066500984 +0100
+++ gpm-1.20.1/src/synaptics.c	2005-02-07 11:41:56.014819824 +0100
@@ -149,6 +149,12 @@
 **                                                                                 
 **/
 
+double ceil (double x) {
+	return x;
+}
+float ceilf (float x) {
+       return x;
+}
 
 /*
 ** TODO
